muskathlon/models/event_registration.py

```diff
@@ -67,13 +67,14 @@ def _compute_step2_tasks(self):
         for reg in self:
             reg.passport_uploaded = muskathlon_passport in reg.completed_task_ids
 
-    def _compute_amount_raised(self):
+    def _compute_amount_raised(self, force_compute=False):
```

Comment:
I think `force_compute` is no longer used, you could remove it.

Reply:
Oh yes indeed